packages/terra-collapsible-menu-view/src/terra-dev-site/test/collapsible-menu-view/DefaultCollapsibleMenuView.test.jsx

```diff
@@ -51,6 +51,7 @@ class CollapsibleMenuViewDemo extends React.Component {
           shouldCloseOnClick={false}
           onChange={this.handleToggleTwoOnChange}
           isSelected={this.state.toggle2Selection}
+          isSelectable
```

Comment:
Out of curiosity, why did we need this change? Does `Toggle Item 1` also need this change?

Reply:
forgot to remove before commit 🙆‍♂updated : 6a5829b5401cbcb031978cb8b05a4b73acd13af3